doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -261,7 +261,14 @@ Debug samples
 DECT NR+ samples
 ----------------
 
-|no_changes_yet_note|
+* :ref:`dect_shell_application` sample:
+
+  * Added:
+
+    * A brief MAC-level sample on top of DECT PHY interface with new commands to create a periodic cluster beacon, scan for it, associate or disassociate a PT/client, and send data to a FT/beacon random access window.
```

Comment:
I would mention the added command name (`dect mac`) somewhere.

Reply:
added